config/routes.rb

```diff
@@ -54,28 +54,25 @@
   get '/sso' => 'single_sign_on#sso'
 
   # Legacy
-  get '/discover/blog', to: redirect('/discover/updates')
-  get '/explore', to: redirect('/discover')
-  get '/blog',    to: redirect('http://blog.assembly.com')
-
-  # Pages
-  get '/home'             => 'pages#home',        as: :home
-  get '/about'            => 'pages#about',       as: :about
-  get '/terms'            => 'pages#tos',         as: :tos
-  get '/core-team'        => 'pages#core_team',   as: :core_team
-  get '/badges'           => 'pages#badges',      as: :badges
-  get '/pitchweek'        => 'pages#pitch_week',  as: :pitch_week
-  get '/sabbaticals'      => 'pages#sabbaticals', as: :sabbaticals
-  get '/activity'         => 'activity#index',    as: :activity
+  get '/discover/blog' => redirect('/discover/updates')
+  get '/explore' => redirect('/discover')
+  get '/blog'    => redirect('http://blog.assembly.com')
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.